### DISCUSSION TITLE
Discounts on Azure Subscriptions

### DISCUSSION THREAD

**Discussion by arnelo97** [mivano/azure-cost-cli#18] [Ideas]
Is it possible to show discounts on the ouput?

**Comment by mivano** [comment]
I will have to check how to retrieve those with the APIs available. Will give it a try.

**Reply by apex944** [reply]
I'm guessing this discount info was not available via the API ? 
Any chance MS updated the API and a year later it is available?

**Reply by mivano** [reply]
Sorry, this was placed on a backburner and I have not looked into it further.
